typescript/cli/src/config/ism.ts

```diff
@@ -300,3 +320,7 @@ export async function createRoutingConfig(
     domains: domainsMap,
   };
 }
+
+async function createDefaultIsmConfig(context: CommandContext) {
+  return await createTrustedRelayerConfig(context);
+}
```

Comment:
nit: unnecessary await :P
also given this is just 1 line with no transform I would skip this indirection

Reply:
the final straw lmao: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/861af3197c75a9e2f864e03fc4d66271d955827f/.eslintrc#L26

Reply:
removed extra function